ctools/generate_fragmented_sharded_collection.py

```diff
@@ -247,6 +249,7 @@ async def safe_write_chunks(shard, chunks_subset, progress):
     print('Writing collection entry')
     coll_obj = {
             '_id': args.ns,
+            'lastmodEpoch': epoch,
```

Comment:
Not sure why I removed it when I added support for `5.0`. It turned out that entries in `config.collections`  contain `lastmodEpoch` even in versions grater or equal to `5.0`